File: frontend/src/core/models/IncomeModel/IncomeResponse.ts
export class Income {
  id?: number;
  source: string;
  description: string;
  amount: number;
  date: Date;

  constructor(id:number,source:string,description:string,amount:number,date:Date){
    this.id = id;
    this.source=source;
    this.description=description;
    this.amount=amount;
    this.date=date;
  }
}

File: backend/src/main/java/com/project/MoneyT/dto/incomeDTO/IncomeResponse.java
package com.project.MoneyT.dto.incomeDTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class IncomeResponse {
    private Long id;
    private String source;
    private String description;
    private BigDecimal amount;
    private LocalDate date;
    private String createdAt;
}
